docs/source/getting_started.rst

```diff
@@ -1,4 +1,25 @@
 Getting Started
 ===============
 
-This page details how to get started with mdaencore. 
+Installation
+------------
+
+`mdaencore` is currently only installable from source.
+First download and navigate to the source code from GitHub with
+
+.. code-block:: sh
+
+	git clone --depth=1 https://github.com/MDAnalysis/mdaencore.git
+	cd mdaencore
+
+The packages is now installable with `pip`:
+
+.. code-block:: sh
+
+	pip install .
```

Comment:
This is currently missing the additional conda/pip instructions, e.g. recommending conda and installing Cython first. Maybe it'd be easier to just link to the README here, or move the installation from source instructions here?

Reply:
trying these commands in a fresh environment seems to work just fine.